Source/DafnyDriver/Commands/TestCommand.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Linq;
+
+namespace Microsoft.Dafny; 
+
+public class TestCommand : ICommandSpec {
+  public IEnumerable<IOptionSpec> Options =>
+    new IOptionSpec[] {
+      TargetOption.Instance,
+      NoVerifyOption.Instance,
+      EnforceDeterminismOption.Instance,
+      VerificationTimeLimitOption.Instance,
+    }.Concat(ICommandSpec.CommonOptions);
+
+  public Command Create() {
+    var result = new Command("test", "(Experimental) Execute every method in the program that's annotated with the {:test} attribute.");
+    result.AddArgument(ICommandSpec.FilesArgument);
```

Comment:
And also differentiate 'test' and 'generate-tests'

Reply:
> Is it possible to tweak FilesArgument so that we get verify [<file>...] and so on, except for run <file>?

I'm pretty sure it is although it involves tweaking the help generation which I haven't played around with yet.